ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -269,4 +274,30 @@ static boolean bug1642386Affected(Map<String, String> attributes) {
         && attributes.get(Attribute.APP_VERSION) != null
         && attributes.get(Attribute.APP_VERSION).matches("^([0-9]|[0-2][0-7])\\..*"); // <= 27
   }
+
+  // See bug 1673433 for discussion of affected URIs
+  private static boolean bug1673433Affected(String uri, Map<String, String> attributes) {
+    // See: https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-ingestion/
+    // k8s/charts/data-ingestion/templates/filter-configmap.yaml#L14-L41
+    if (uri == null) {
+      return false;
+    }
+
+    final ImmutableSet<String> allowedDocTypes = ImmutableSet.of("/main/", "/core/", "/sync/",
+        "/sslreports", "/FirefoxOS/");
```

Comment:
If I recall correctly, we now catch the "sslreports" and "FirefoxOS" cases as unwanted data in other places within MessageScrubber. I worry now that we're adding confusion by explicitly allowing these here, but then catching them separately elsewhere.

It's probably the right thing to separate these out for having the right bug attributed. But I think it warrants some additional commentary in the code. I'd like to see commentary on this object specifically to call out that `sslreports` and `FirefoxOS` are thrown out elsewhere, but allowed here for historical consistency.

Reply:
> This logic looks correct. Verifying it is exhausting. Every time I have read https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-ingestion/k8s/charts/data-ingestion/templates/filter-configmap.yaml#L14-L41 I have to read it over at least five times to convince myself I've properly parsed the negations and nils.

I [commented](https://bugzilla.mozilla.org/show_bug.cgi?id=1673433#c4) that another option here is to avoid translating this logic wholesale and instead re implement the still relevant pieces (that haven't already been implemented) as-needed by observing the garbage we begin to receive when removing the edge filter.

As-is this logic should be a noop on URIs that made it to `payload_bytes_raw`.

Reply:
+1 to only re-implementing the pieces we actually need. This will hopefully simplify the logic and keep it more maintainable in the future.